api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -57,7 +60,7 @@ public UserController(
       Provider<WorkbenchConfig> configProvider,
       FireCloudService fireCloudService,
       UserService userService,
-      Provider<Cloudbilling> cloudBillingProvider) {
+      @Qualifier(USER_PROXY_CLOUD_BILLING) Provider<Cloudbilling> cloudBillingProvider) {
```

Comment:
Why the `PROXY` language here? Elsewhere we have `END_USER_FOO` as a token which seems accurate to me. Is there a semantic difference?